portfolio/src/main/java/com/google/sps/data/Data.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.data;
+
+/** Represents UFO sightings at a specific lat lng point. */
+public class Data {
```

Comment:
maybe UfoLocation? Or UfoData if you plan on adding more info later? "Data" is very vague

Reply:
Change to UfoData, done